frontend/src/components/Jobs.vue

```diff
@@ -106,26 +138,40 @@ export default {
     fetchData() {
       return fetchCctrayJson()
           .then(this.marshalData)
-          .then((data) => this.jobs = data)
+          .then((data) => {
+            this.visibleJobs = data.filter(j => !preferences.isJobHidden(j.name))
+            this.hiddenJobs = data.filter(j => preferences.isJobHidden(j.name))
+          })
           .catch((reason) => {
             console.error(reason);
             return Promise.reject(reason);
           });
+    },
+    toggleHiddenJobs() {
+      this.showHiddenJobs = !this.showHiddenJobs
     }
   }
 }
 </script>
 
 <style scoped>
 
-#jobs {
+.jobs {
   font-family: "OpenSans", sans-serif;
   display: grid;
   align-items: center;
   grid-template-rows: repeat(auto-fit, minmax(90px, 115px));
   grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
   grid-gap: 10px;
   width: 100%;
+  margin-bottom: 3em;
+}
+
+.hidden_jobs {
+  font-weight: bold;
+  color: #333;
```

Comment:
Do we really need this styling for hidden jobs? as hidden jobs are also jobs. According to me it should have the same style as normal jobs

Reply:
This is the styling for the headline "5 hidden jobs". This does not exist for normal jobs, so thats why we have this styling here.

Reply:
ooh my bad, I thought it was for all the fonts (not just header). I'll mark this comment as resolve